pkg/sidecarproxy/server.go

```diff
@@ -0,0 +1,108 @@
+package sidecarproxy
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/v3io/sidecar-proxy/pkg/sidecarproxy/common"
+	"github.com/v3io/sidecar-proxy/pkg/sidecarproxy/metricshandler"
+	"github.com/v3io/sidecar-proxy/pkg/sidecarproxy/metricshandler/jupyterkernelbusyness"
+	"github.com/v3io/sidecar-proxy/pkg/sidecarproxy/metricshandler/numofrequests"
+
+	"github.com/nuclio/logger"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+type Server struct {
+	logger          logger.Logger
+	listenAddress   string
+	forwardAddress  string
+	metricsHandlers []metricshandler.MetricHandler
+}
+
+func NewProxyServer(logger logger.Logger,
```

Comment:
Should probably be `NewServer` since the struct is not called `ProxyServer` (or the other way around)